### ISSUE TITLE
Issue while integrating with QnA

### ISSUE THREAD

**Issue by VijayaReddipalli** [microsoft/BotFramework-Composer#2556]
Not able to fetch reponse form QnA. 
Instead I'm getting "Object reference not set to an instance of an object."

<img width="891" alt="qnaresponse" src="https://user-images.githubusercontent.com/63333402/78747639-83fa8280-7987-11ea-9486-0679e7bbcd7f.PNG">

![image](https://user-images.githubusercontent.com/63333402/78748145-e30cc700-7988-11ea-9e3c-65f47b47f259.png)

<img width="147" alt="settings" src="https://user-images.githubusercontent.com/63333402/78748294-48f94e80-7989-11ea-86e5-f9711d56479f.PNG">

Please check and let me know ASAP.

**Comment by luhan2017** [comment]
It seems like you didn't fill some field, could you scroll down the form editor to see whether there is any properties need to be filled?

**Comment by VijayaReddipalli** [comment]
These are the values I have given in properties panel.

![image](https://user-images.githubusercontent.com/63333402/78748922-ca051580-798a-11ea-9a7e-44fa30570370.png)


![image](https://user-images.githubusercontent.com/63333402/78748892-c07bad80-798a-11ea-848c-37ff5af292cd.png)

**Comment by VijayaReddipalli** [comment]
let me know if there is anything needed to be updated.

**Comment by luhan2017** [comment]
@VijayaReddipalli , I just found a bug in our schema file, which I already have a PR(https://github.com/microsoft/botbuilder-dotnet/pull/3704) to fix this. One mitigation could be, you just go to your dialog file and check whether the "cardNoMatchResponse " property have an extra blank there, you can remove that blank and try again.

**Comment by VijayaReddipalli** [comment]
@luhan2017  , I have checked but there is no extra space.I'm still facing the same issue.

**Comment by VijayaReddipalli** [comment]
Please let me know once the bug has been fixed.

**Comment by luhan2017** [comment]
@VijayaReddipalli which version of composer are you using? I tried in latest master, it works fine for me. I just clone the QnAMastakerLuisSample, fill in the QnA settings and start in emulator. here is what I got:

![image](https://user-images.githubusercontent.com/32191031/78764552-4c0c3300-79b9-11ea-81c7-5c2c48683af6.png)

![image](https://user-images.githubusercontent.com/32191031/78764609-61815d00-79b9-11ea-8d43-406825fb8c95.png)
